src/components/Experiment/Experiment.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {
+  Experiment as OptimizeExperiment,
+  ExperimentProps as OptimizeExperimentProps,
+} from 'react-optimize';
+
+export enum ExperimentID {
```

Comment:
Can you maybe take the content that you put in the PR description and add it as a comment in this file so that it's preserved along with the code?